doc/use_cases/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Use Cases"
+---
+
+## Overview
+
+OpenTitan's mission is to raise the security bar industry wide by implementing a
```

Comment:
nit: should industry-wide be hyphenated?

Reply:
Done.

Reply:
Thx